internal/command/postgres/add_flycast.go

```diff
@@ -0,0 +1,148 @@
+package postgres
+
+import (
+	"context"
+	"fmt"
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/spf13/cobra"
+	"github.com/superfly/flyctl/api"
+	"github.com/superfly/flyctl/client"
+	"github.com/superfly/flyctl/internal/app"
+	"github.com/superfly/flyctl/internal/command"
+	"github.com/superfly/flyctl/internal/command/apps"
+	"github.com/superfly/flyctl/internal/flag"
+	mach "github.com/superfly/flyctl/internal/machine"
+)
+
+func newAddFlycast() *cobra.Command {
+	const (
+		short = "Switch from DNS to flycast based pg connections"
+		long  = short + "\n"
+
+		usage = "add_flycast"
+	)
+
+	cmd := command.New(usage, short, long, runAddFlycast,
+		command.RequireSession,
+		command.RequireAppName,
+	)
+
+	flag.Add(
+		cmd,
+		flag.App(),
+		flag.AppConfig(),
+	)
+
+	cmd.Hidden = true
+
+	return cmd
+}
+
+func runAddFlycast(ctx context.Context) error {
+	var (
+		client  = client.FromContext(ctx).API()
+		appName = app.NameFromContext(ctx)
+	)
+
+	app, err := client.GetAppCompact(ctx, appName)
+	if err != nil {
+		return fmt.Errorf("failed retrieving app %s: %w", appName, err)
+	}
+
+	if !app.IsPostgresApp() {
+		return fmt.Errorf("app %s is not a postgres app", appName)
+	}
+
+	ctx, err = apps.BuildContext(ctx, app)
+	if err != nil {
+		return err
+	}
+
+	switch app.PlatformVersion {
+	case "machines":
+		err := doAddFlycast(ctx)
+		if err != nil {
+			return err
+		}
+
+		fmt.Println("Flycast added!")
```

Comment:
We should be using Fprintln here:
Reference: https://github.com/superfly/flyctl/blob/master/internal/command/postgres/attach.go#L281